docs/custom-model-integration/Dockerfile.reference

```diff
@@ -0,0 +1,39 @@
+FROM python:3.10-slim@sha256:684b1aaf96a7942b3c3af438d162e0baa3510aa7af25ad76d238e0c746bdec79
+
+# Specify the repository source URL for reference and access in Kaito packages.
+LABEL org.opencontainers.image.source=https://github.com/azure/kaito
+
+# ARG WEIGHTS_PATH
+ARG MODEL_TYPE
+ARG VERSION
+
+# Set the working directory
+WORKDIR /workspace/tfs
+
+# Write the version to a file
+RUN echo $VERSION > /workspace/tfs/version.txt
+
+# First, copy just the preset files and install dependencies
+# This is done before copying the code to utilize Docker's layer caching and
+# avoid reinstalling dependencies unless the requirements file changes.
+# Inference
+COPY presets/inference/${MODEL_TYPE}/requirements.txt /workspace/tfs/inference-requirements.txt
+RUN pip install --no-cache-dir -r inference-requirements.txt
+
+COPY presets/inference/${MODEL_TYPE}/inference_api.py /workspace/tfs/inference_api.py
+
+# Fine Tuning
+COPY presets/tuning/${MODEL_TYPE}/requirements.txt /workspace/tfs/tuning-requirements.txt
+RUN pip install --no-cache-dir -r tuning-requirements.txt
+
+COPY presets/tuning/${MODEL_TYPE}/cli.py /workspace/tfs/cli.py
+COPY presets/tuning/${MODEL_TYPE}/fine_tuning.py /workspace/tfs/fine_tuning.py
+COPY presets/tuning/${MODEL_TYPE}/parser.py /workspace/tfs/parser.py
+COPY presets/tuning/${MODEL_TYPE}/dataset.py /workspace/tfs/dataset.py
+
+# Copy the metrics server
+COPY presets/tuning/${MODEL_TYPE}/metrics/metrics_server.py /workspace/tfs/metrics_server.py
+
+# Remove Weights Copying
```

Comment:
Why do we need to copy weight file here?

Reply:
This was commented out

Reply:
Removed